audit.go

```diff
@@ -599,16 +599,17 @@ var sizeofAuditStatus = int(unsafe.Sizeof(AuditStatus{}))
 // between the kernel and user-space.
 // https://github.com/linux-audit/audit-kernel/blob/v4.7/include/uapi/linux/audit.h#L413-L427
```

Comment:
Please update this to https://github.com/linux-audit/audit-kernel/blob/v5.9/include/uapi/linux/audit.h#L457-L474 which is the first version supporting this field.

Reply:
Thanks, updated